src/read/elf/file.rs

```diff
@@ -173,6 +173,24 @@ where
     ) -> Option<ElfSection<'data, 'file, Elf, R>> {
         None
     }
+
+    fn elf_verneed_by_version(&self, version: VersionIndex) -> read::Result<Option<(&'data elf::Verneed<Elf::Endian>, SectionIndex)>> {
+        let svt = self.sections.versions(self.endian, self.data)?;
+        if let Some(svt) = svt {
+            if let Some(ver) = svt.version(version)? {
+                if let Some((mut verneed_iter, section_index)) = self.sections.gnu_verneed(self.endian, self.data)? {
+                    while let Some((verneed, mut vernaux_iter)) = verneed_iter.next()? {
```

Comment:
Parsing all these verneed entries again for every import isn't ideal. We already parsed them once when building `svt`. The problem is we didn't store the parsed info in `svt`. We can probably add another field in [`Version`](https://github.com/gimli-rs/object/blob/2f45bc0e1b82df1229453ceed1361b8558b8529b/src/read/elf/version.rs#L38).